pkg/inventory/inventorycm.go

```diff
@@ -6,7 +6,7 @@
 // ConfigMap resource which stores the set of inventory
 // (object metadata).
 
-package prune
+package inventory
```

Comment:
I think we should keep this in a package separate from both prune and inventory. This is just one of the (possibly many) implementations of the inventory.

Reply:
I think this properly belongs in the `inventory` package, but I'm not wedded to it. Please let me know what other package this could be put in.

Reply:
I just think it is useful to keep the implementation separate from the interface. This is just the implementation of the `Inventory` interface using a configmap, but we will have an implementation with ResourceGroup that must live in a package in another repo.